src/Docker/docker-compose.test-local.yml

```diff
@@ -65,10 +65,12 @@ services:
       - AZURE_STORAGE_CONNECTION_STRING='DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://blob-storage:10000/devstoreaccount1;'
       - EMAIL_PASSWORD=$EMAIL_PASSWORD
       - ADMIN_EMAIL=<your_email>
+      - AUTH0_CLIENT_ID = 'sQPoZzmH4QaAHVEJrDaK3pPeHG0SmCtr'
+      - AUTH0_CLIENT_SECRET = $AUTH0_CLIENT_SECRET
+      - REVIEWER_EMAIL_LIST = <list of reviewer emails>
```

Comment:
These will need to be added to the `docker-compose.yml` file too, with reviewer_email_list as `vectoratlas@icipe.org` for now